dev/enc28j60/enc28j60-ip64-driver.c

```diff
@@ -85,12 +85,14 @@ PROCESS_THREAD(enc28j60_ip64_driver_process, ev, data)
   PROCESS_BEGIN();
 
   while(1) {
-    etimer_set(&e, 1);
-    PROCESS_WAIT_EVENT();
     len = enc28j60_read(ip64_packet_buffer, ip64_packet_buffer_maxlen);
     if(len > 0) {
+      etimer_set(&e, (len + 15) >> 4);
```

Comment:
It is explained in the commit message:
30b3041e72b48b314442685adbf1a09a08b8bdfc